### PR TITLE
chore(c*): backfill `.repo-metadata.json` files

### PR DESCRIPTION
**Pull request by coryan**
Part of the work for #11519

- - -
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/11596)

### CODE REVIEW

File: google/cloud/composer/v1/.repo-metadata.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "composer.googleapis.com",
+    "api_shortname": "composer",
+    "client_documentation": "https://cloud.google.com/cpp/reference/composer/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/savedsearches/5249404r",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Cloud Composer API",
+    "product_documentation": "https://cloud.google.com/composer/docs",
```
Comment (dbolduc) [inline google/cloud/composer/v1/.repo-metadata.json:10]: nit: "https://cloud.google.com/composer" is consistent with the others

File: google/cloud/connectors/v1/.repo-metadata.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "connectors.googleapis.com",
+    "api_shortname": "connectors",
+    "client_documentation": "https://cloud.google.com/cpp/reference/connectors/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/savedsearches/5929031",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Connectors API",
```
Comment (dbolduc) [inline google/cloud/connectors/v1/.repo-metadata.json:9]: maybe: "Apigee Connectors API"

The branding list does not endorse "Connectors API" or "Apigee Connectors API", FWIW. The closest thing that shows up in the list is "Apigee Integration", but eh.

File: google/cloud/contactcenterinsights/v1/.repo-metadata.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "contactcenterinsights.googleapis.com",
+    "api_shortname": "contactcenterinsights",
+    "client_documentation": "https://cloud.google.com/cpp/reference/contactcenterinsights/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/savedsearches/6143399",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Contact Center AI Insights API",
+    "product_documentation": "https://cloud.google.com/contact-center/insights/docs",
```
Comment (dbolduc) [inline google/cloud/contactcenterinsights/v1/.repo-metadata.json:10]: nit: "https://cloud.google.com/solutions/ccai-insights" would be more consistent

File: google/cloud/container/v1/.repo-metadata.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "container.googleapis.com",
+    "api_shortname": "container",
+    "client_documentation": "https://cloud.google.com/cpp/reference/container/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/savedsearches/559746",
+    "language": "cpp",
+    "library_type": "GAPIC_AUTO",
+    "name_pretty": "Kubernetes Engine API",
+    "product_documentation": "https://cloud.google.com/kubernetes-engine/docs/",
```
Comment (dbolduc) [inline google/cloud/container/v1/.repo-metadata.json:10]: nit: "https://cloud.google.com/kubernetes-engine/" would be more consistent

File: google/cloud/containeranalysis/v1/.repo-metadata.json [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "containeranalysis.googleapis.com",
+    "api_shortname": "containeranalysis",
+    "client_documentation": "https://cloud.google.com/cpp/reference/containeranalysis/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:187156",
```
Comment (dbolduc) [inline google/cloud/containeranalysis/v1/.repo-metadata.json:6]: comment: I guess this component is shared by `artifactregistry` :shrug:

**Review by dbolduc — APPROVED** [review]

**Review by dbolduc — APPROVED** [review]

File: google/cloud/composer/v1/.repo-metadata.json (hunk 1 above)
Comment (coryan) [reply google/cloud/composer/v1/.repo-metadata.json:10]: "the others"... I assume you mean other .repo-metadata.json files, but not the README files:

https://github.com/googleapis/google-cloud-cpp/blob/83f7fad396b6cd4bf40673d109d9aed9f21629af/google/cloud/composer/README.md?plain=1#L60

And when available we will be using the YAML files as the source of truth.  And those seem to use the `/docs` link:

https://github.com/googleapis/googleapis/blob/ef2e2ea532248d6dc40a56bc6c95cea858ba31b6/google/cloud/workstations/v1/workstations_v1.yaml#L113

I think you have an opinion about where these links should point to.  I cannot say that your opinion is wrong, but I am not sure that opinion matches our READMEs, the existing practices for other languages, or any other guidance.

I am going to leave these alone.  Once all the `.repo-metadata.json` files are in you are welcome to make them consistent with each other, but I think getting them to be automatically updated is really the goal.

File: google/cloud/connectors/v1/.repo-metadata.json (hunk 2 above)
Comment (coryan) [reply google/cloud/connectors/v1/.repo-metadata.json:9]: I think this is too low level for anybody to care about branding.  In any case:

https://github.com/googleapis/googleapis/blob/ef2e2ea532248d6dc40a56bc6c95cea858ba31b6/google/cloud/connectors/v1/connectors_v1.yaml#L4

File: google/cloud/contactcenterinsights/v1/.repo-metadata.json (hunk 3 above)
Comment (coryan) [reply google/cloud/contactcenterinsights/v1/.repo-metadata.json:10]: See above.

File: google/cloud/containeranalysis/v1/.repo-metadata.json (hunk 5 above)
Comment (coryan) [reply google/cloud/containeranalysis/v1/.repo-metadata.json:6]: yes, they do.

File: google/cloud/composer/v1/.repo-metadata.json (hunk 1 above)
Comment (dbolduc) [reply google/cloud/composer/v1/.repo-metadata.json:10]: My opinion is only that all or none should use the `/docs` link. It seemed like the pattern was to have none before this PR.

How much do I care: 0

**Pull request merged**